miden-lib/asm/wallets/basic.masm

```diff
@@ -0,0 +1,29 @@
+use.miden::sat::account
+use.miden::sat::tx
+
+#! This procedure adds the given asset to the undelying account.
+#! 
+#! Inputs: [ASSET]
+#! Outputs: [0, 0, 0, 0, ...]
+#!
+#! - ASSET is the asset to be received, can be fungible or non-fungible
+export.receive_asset
+    exec.account::add_asset
+    padw swapw dropw
+end
+
+#! Creates a note which sends a single asset to the specified recipient.
+#!
+#! Inputs: [ASSET, tag, RECIPIENT, ...]
+#! Outputs: [note_ptr, 0, 0, 0, 0, 0, 0, 0, 0, ...]
+#!
+#! - ASSET is the non-fungible asset of interest.
+#! - tag is the tag to be included in the note.
+#! - RECIPIENT is the recipient of the note.
+#! - note_ptr is the pointer to the memory address in the kernel.
+#!   This cannot directly be accessed from another context. 
+export.send_asset
+    exec.account::remove_asset
+    exec.tx::create_note
+    swapdw movup.8
```

Comment:
I'd probably change the code to something similar to what I described as "approach 1" in https://github.com/0xPolygonMiden/miden-base/issues/234#issuecomment-1714479410.

Reply:
done